cluster/operator-install.sh

```diff
@@ -16,7 +16,7 @@
 
 set -ex
 
-export VERSION=latest
+export VERSION=v0.0.0-1
```

Comment:
Why not to use operator version -> v0.0.1?

Reply:
I wanted to make sure that the version built for the tests is different than one that we would release.

Reply:
What about using v0.0.0 like in go.mod when we relay on latest?

Reply:
:+1: